src/components/HeadCellMenu/MenuList/__tests__/MenuGroup.test.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { fireEvent, render, screen } from '@testing-library/react';
+import MenuGroup, { interceptClickOnMenuItems } from '../MenuGroup';
+import { MenuItemGroup } from '../../types';
+
+describe('MenuGroup', () => {
+  describe('interceptClickOnMenuItems', () => {
```

Comment:
Could this be tested via the HeaderCellMenu component instead?

Reply:
Or even the `MenuGroup` component?

Reply:
I added extended this test to invoke a mocked function which i pass into `interceptClickOnMenuItems`. So now we are sure that it's been called